docs/current/data-model-howto.md

```diff
@@ -168,47 +167,107 @@ Boots exist as individual documents under the `boot` subdirectory. They are refe
     └── etcd-proxy.yml
 ```
 
+### Workflows
+
+Workflows exist as individual documents under the `workflows` subdirectory. They are referenced by a self-assigned ID described in the document:
+
+```
+.
+└── workflows
+    ├── clevos-accessor.yml
+    ├── k8s-master.yml
+    └── k8s-node.yml
+```
 
 ### Sites
 
-Sites are represented by individual subdirectories. One directory == one site == one managed group of machines. Each SoT can contain multiple sites. Each of these sites shares the same assets/boot/os configuration files.
+Sites are represented by individual subdirectories under the top level `sites` directory. Each `site` under the `sites` directory corresponds to a managed group
+of machines.
 
-Each site has _at least_ two documents, the specially named `env.yml` and at least one document describing an inventory of hosts. You may use YAML's triple-dash `---` separator to combine multiple inventory documents into one file.
+Each site has _at least_ two documents, the specially named `env.yml` and at least one document describing an inventory of hosts, a cluster. You may use YAML's triple-dash `---` separator to combine multiple inventory documents into one cluster file.
```

Comment:
This whole section is a little awkward, since you are updating it, how about:
"Each site requires one file named `env.yml`, all other files will be treated as a cluster definition that describe their host inventory. You may use...."

Reply:
Will do.